api/tests/unit/organisations/test_unit_organisations_views.py

```diff
@@ -657,6 +659,44 @@ def test_when_subscription_is_set_to_non_renewing_then_cancellation_date_set_and
     mocked_cancel_chargebee_subscription.assert_not_called()
 
 
+@pytest.mark.freeze_time("2023-01-19T09:09:47.325132+00:00")
+@mock.patch("organisations.models.cancel_chargebee_subscription")
+def test_when_subscription_is_set_to_non_renewing_then_cancellation_date_set_and_current_term_end_is_missing(
+    mocked_cancel_chargebee_subscription: MagicMock,
+    subscription: Subscription,
```

Comment:
Shouldn't we use e.g. `chargebee_subscription` here?

Reply:
Sure that's a good idea, I've added it.